client/src/app/pages/applications/components/application-list-expanded-area/application-list-expanded-area-extras.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { TableComposable, Thead, IRow, Tr, Th, Tbody, Td } from '@patternfly/react-table';
+
+import {
+  DescriptionList,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+} from "@patternfly/react-core";
+
+import { Application, Task } from "@app/api/models";
+import { ApplicationTags } from "../application-tags";
+import { ApplicationAnalysisStatus } from "../../components/application-analysis-status";
+
+export interface IApplicationListExpandedAreaProps {
+  application: Application;
+  tasks: Task[];
+}
+
+export const ApplicationListExpandedAreaExtras: React.FC<
+  IApplicationListExpandedAreaProps
+> = ({ application, tasks }) => {
+  const { t } = useTranslation();
+
+  const rows: IRow[] = [];
+  tasks?.forEach((task) => {
+    rows.push(
+      <Tr key="Name">
+        <Td dataLabel="Addon">{task.addon}</Td>
+        <Td dataLabel="Status">
+          <ApplicationAnalysisStatus state={task.state? task.state : "No task"} />
+        </Td>
+      </Tr>
+    )
+  })
+
+  var tasksGroup;
```

Comment:
Using 'let' is preferable here - It is just an updated block scoped form of var. We typically use either let or const for defining variables. Minor nitpick for consistency.